src/config/dark-sites.config

```diff
@@ -143,6 +143,8 @@ clash3d.com
 clashofstats.com
 clt.gg
 codepen.io
+codeiguess.github.io
```

Comment:
![image](https://user-images.githubusercontent.com/25481501/133978771-d2079610-1c89-4b80-9c14-84a40ee820a0.png)

Seems to be not dark.

Reply:
Sorry mate, site's not up yet.

Reply:
Then why should I merge this PR?

Reply:
codeiguess.github.io/Notepad is dark mode and standalone. It would be nice to add it. codeiguess.github.io itself isn't up yet, however.

Reply:
Then let's merge /notepad for now

Reply:
Please remove the other entry.